tests/example.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test';
```

Comment:
Can you rename the file to `home.spec.ts` of `list.spec.ts` , so it doesnt appear to be a sample?